editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -108,30 +116,49 @@ const VerticalFeatureTabs: React.FC<{ features: Feature[] }> = ({
                     title={"Copy from"}
                     labelId={`select-label-${i}`}
                     value={""}
-                    onChange={() => console.log(`TODO - Copy data from another tab`)}
+                    onChange={() =>
+                      console.log(`TODO - Copy data from another tab`)
+                    }
                     name={""}
                     style={{ width: "200px" }}
                   >
-                    {features.filter((feature) => feature.properties?.label !== activeTab).map((option) => (
-                      <MenuItem
-                        key={option.properties?.label}
-                        value={option.properties?.label}
-                      >
-                        {`${schema.type} ${option.properties?.label}`}
-                      </MenuItem>
-                    ))}
+                    {features
+                      .filter(
+                        (feature) => feature.properties?.label !== activeTab,
+                      )
+                      .map((option) => (
+                        <MenuItem
+                          key={option.properties?.label}
+                          value={option.properties?.label}
+                        >
+                          {`${schema.type} ${option.properties?.label}`}
+                        </MenuItem>
+                      ))}
                   </SelectInput>
                 </InputLabel>
               </Box>
             </Box>
             <SchemaFields
+              sx={(theme) => ({
+                display: "flex",
+                flexDirection: "column",
+                gap: theme.spacing(2),
+              })}
```

Comment:
This is the only change in this file, the rest is just linting (need to resolve this sooner rather than later!)